tests/test_ecs.py

```diff
@@ -185,3 +185,57 @@ def test_container_instance_resource(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_container_instance_update_agent(self):
+        session_factory = self.replay_flight_data('test_ecs_container_instance_update_agent')
+        p = self.load_policy(
+            {'name': 'container-instance-update-agent',
+             'resource': 'ecs-container-instance',
+             'actions':[
+                 {'type': 'update-agent'}
+                 ]},
+             session_factory=session_factory)
+        resources = p.run()
+        client = session_factory().client('ecs')
+        c_instances = client.list_container_instances(
+                cluster='default').get('containerInstanceArns')
+        version = client.describe_container_instances(cluster='default',
+                containerInstances=c_instances)['containerInstances'][0]['versionInfo']['agentVersion']
+        self.assertEqual(version, '1.17.0')
+
+    def test_container_instance_drain(self):
+        session_factory = self.replay_flight_data('test_ecs_container_instance_drain')
+        p = self.load_policy(
+            {'name': 'container-instance-update-agent',
+             'resource': 'ecs-container-instance',
+             'actions':[
+                 {'type': 'update-state',
+                     'state': 'DRAINING'}
+                 ]},
+             session_factory=session_factory)
+        resources = p.run()
+        client = session_factory().client('ecs')
+        c_instances = client.list_container_instances(
```

Comment:
could you omit this call and just use the Arn from resources to describe the container instances?